src/main.ts

```diff
@@ -14,7 +14,7 @@ Vue.use(VueCompositionApi)
 Vue.use(CoolLightBox)
 
 Vue.use(VueGtag, {
-  config: { id: 'UA-154739393-1' },
+  config: { id: 'UA-132228705-11' },
   enabled: process.env.NODE_ENV === 'production'
```

Comment:
Turns out this line is the reason why it not work.
What is the `process.env.NODE_ENV` for the development site?
@tai271828

Reply:
@Neilxx ah, I did not assign the value of `process.env.NODE_ENV` in `.env`. Do you happen to know if I should set the value in `.env`?